src/Adapter/ORM/DoctrineORMAdapter.php

```diff
@@ -22,7 +22,7 @@ public function recomputeChangeSet(BaseLifecycleEventArgs $event): void
     {
         $object = $event->getObject();
 
-        $em = $event->getEntityManager();
+        $em = $event->getObjectManager();
```

Comment:
This change is about a different issue, it should be in another PR

Reply:
You are right. How we can move it to the #1393 PR?